tokens/properties/components/InputExtender.json

```diff
@@ -39,6 +39,23 @@
                 "vertical": {
                     "value": "{dimension.spacing.xsmall.value}"
                 }
+            },
+            "placeholder": {
```

Comment:
I'm not sure `placeholder` is the ideal term here as I feel that this is already a fixed term that refers to the literal placeholder in `<input>` HTML elements. (Also given that this component _does_ include such a input element, even though just as a sub-component)

How about calling this `content` instead, as these styles are intentionally applied to the content of the extender? (even though they are overridable by the application)

Reply:
TL;DR: I'll remove the placeholder level and adjust the tokens to make them reflect the default text style of the menu's content.

I hesitated to use the term "placeholder" for the reason you're mentioning. I thought that the object structure ("content-placeholder") would clarify the meaning, but I agree that the name of the variables can still be confusing in their final context (the InputWithExtender.vue file). Probably the whole placeholder approach was wrong, I think we should look at these as the default text styles of the menu content, instead of as some temporary default.